syncer/syncer.go

```diff
@@ -50,9 +28,6 @@ const (
 
 	// the max amount of layer delays syncer can tolerate before it tries to validate a layer.
 	maxHareDelayLayers uint32 = 10
-
-	// the max number of attempts syncer tries to advance the processed layer within a sync run.
-	maxAttemptWithinRun = 3
```

Comment:
do we not need attempts anymore?

Reply:
the reason why we used to have multiple attempts is that we would sync a layer, and then it fails when mesh fails to process the layer. so we would let it try max attempts times to process after we sync the data. now that the two (syncing data and processing ballots) are separated into two goroutines, there is no such need anymore